### DISCUSSION TITLE
Docker Demo - can't open Log Viewer

### DISCUSSION THREAD

**Discussion by iavor-k** [riclolsen/json-scada#155] [Q&A] [answered]
Hello, 
I've installed json-scada on a ubuntu 24. I managed to compile and start the docker demo but I when I open the Log Viewer page, I get the following message:

```
Error: connect ECONNREFUSED 127.0.0.1:6688
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1607:16)
```
It seems that the log-io is not running. Looking in the docker-compose files I can't see any lines related to "log-io". 
I'm not able to create new tags as well, so I have to add them directly via the mongo client. 
Is there some features removed from the docker demo by design or I am missing something?
Thanks!

**Comment by riclolsen** [comment, marked as answer]
Hi!
The docker demo indeed does not include the Log Viewer, this is the case also for install scripts for Linux. I am not sure this solution adopted for Windows is the best for Linux and Docker, probably not. On Linux, Supervisor collects logs and I am thinking about integrating with https://logdy.dev/. This may be a solution for Docker as well. I am open to suggestions.
The problem with tags is however unexpected, sorry about that. 
Anyway this UI for editing tags, connections, users, etc will be replaced by a new upgraded one that is almost done, it will be published ASAP.
Best regards
Ricardo
